Framework/Algorithms/src/ConvertUnitsUsingDetectorTable.cpp

```diff
@@ -91,168 +80,23 @@ void ConvertUnitsUsingDetectorTable::init() {
                   "http://www.mantidproject.org/ConvertUnits).");
 }
```

Comment:
`ConvertUnits::init` uses some validators for `InputWorkspace`. Since `ConvertUnitsUsingDetectorTable` is basically doing the same thing, maybe it makes sense to also put the stricter validators here?